DESCRIPTION

```diff
@@ -46,3 +47,4 @@ License: LGPL-3 | file LICENSE
 RoxygenNote: 7.1.0
 Suggests: testthat
 ValidationKey: 317207214
+
```

Comment:
Not sure if this is the case but since this commit building magpie4 is not possible. Could be related to a trailing blank line.
Probably related to
https://github.com/r-lib/devtools/issues/383
https://github.com/r-lib/devtools/issues/394

Additionally, please refrain from merging your own PRs

Reply:
lucode2::buildLibrary showed no issues for me. There was only one warning that pointed to a called library(lucode) in the examples sections. Somewhere in the functions

Reply:
it was indeed the blank line which was causing the problem. I pushed a fix to the repo